modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/play26/Play26CallAdapterFactory.mustache

```diff
@@ -18,8 +18,7 @@ public class Play26CallAdapterFactory extends CallAdapter.Factory {
 
     private Function<RuntimeException, RuntimeException> exceptionConverter = Function.identity();
 
-    public Play26CallAdapterFactory() {
-    }
+    public Play26CallAdapterFactory() { }
```

Comment:
it wasn't that constructor, it was the one below with a parameter :)

Reply:
:+1: